package.json

```diff
@@ -48,13 +43,6 @@
     "warning": "^2.1.0"
   },
   "devDependencies": {
-    "babel-preset-react-native-stage-0": "^1.0.1",
-    "del": "^2.2.0",
-    "fbjs-scripts": "^0.6.0",
-    "gulp": "^3.9.1",
-    "gulp-babel": "^6.1.2",
-    "gulp-flatten": "^0.2.0",
-    "gulp-watch": "^4.3.6",
-    "run-sequence": "^1.1.5"
+    "babel-preset-react-native-stage-0": "^1.0.1"
```

Comment:
since we are not transpiling i think i should remove this as well. expectation is that projects should bring their own babelrc with the right preset.